README.md

```diff
@@ -26,7 +26,7 @@ In this Project, I used HTML and CSS (Media, Transition, flexbox, Grid and more)
 
 Contributions, issues, and feature requests are welcome!
 
-Feel free to check the [Portfolio](https://sediqullahbadakhsh.github.io/badakhsh-portfolio/).
+Feel free to check the [Badakhsh-Portfolio](https://sediqullahbadakhsh.github.io/badakhsh-portfolio/).
 
 ## Show your support
 
```

Comment:
- [ Optional] Kindly use this Readme[ template](https://github.com/microverseinc/readme-template) to update your README.md file and put the deployed into[ Live demo ](url)

Reply:
ok thank you so much, Ill change it right away.